doc/acid/strutil.md

```diff
@@ -0,0 +1,93 @@
+#   Name
+
+acid.strutil
+
+#   Status
+
+This library is considered production ready.
+
+#   Description
+
+It provides with several often used string operation utilities.
+Most of them follows python-style.
+
+#   Synopsis
+
+```lua
+local strutil = require("acid.strutil")
+
+strutil.split('a/b/c', '/')     -- {'a', 'b', 'c'}
+
+
+-- convert series of data into human readable string.
+strutil.to_str(1,2,{10,a=1,20}) -- 12{10,20,a=1}
+```
+
+#   Methods
+
+##  strutil.rsplit
+
+**syntax**:
+`strutil.rsplit(str, pattern, opts)`
+
+Same as `strutil.split` except when `maxsplit` is specified,
+it starts splitting from right to left.
+
+##  strutil.split
+
+**syntax**:
+`strutil.split(str, pattern, opts)`
+
+Split string `str` with separator `pattern`
+
+```
+strutil.split('a/b/c/d', '/', 2)     -- {'a', 'b', 'c/d'}
+```
+
+**arguments**:
+
+-   `str`:
+    is the string to split.
+
+-   `pattern`:
+    is separator in lua string pattern or a plain text string.
+    Depends on the third argument.
```

Comment:
is a separator in lua string patterns or a plain text string depending on the third argument.